k8s/jenkins/chart/jenkins/templates/application.yaml

```diff
@@ -78,13 +78,11 @@ spec:
     matchLabels:
       app.kubernetes.io/name: "{{ .Release.Name }}"
   componentKinds:
-  - group: v1
```

Comment:
Following https://github.com/GoogleCloudPlatform/click-to-deploy/pull/600#discussion_r298624035. Should we restore PVC?